example/glide_computer_lib/CMakeLists.txt

```diff
@@ -25,3 +25,7 @@ cmake_minimum_required(VERSION 3.2)
 add_library(glide_computer_lib STATIC glide_computer_lib.cpp include/glide_computer_lib.h)
 target_link_libraries(glide_computer_lib PRIVATE mp-units::core-fmt PUBLIC mp-units::si mp-units::utility example_utils)
```

Comment:
Should we still link those `mp-unit::XXX` targets when a module target is linked?

Reply:
Yes, because `mp-units::modules` is only linked when it exists.

Reply:
That does sounds more correct.

Reply:
Unfortunately, it's not straight forward.
The tests and examples use macros from the implementation of different headers.